src/main/kotlin/org/rust/lang/core/types/infer/TypeInference.kt

```diff
@@ -102,7 +102,7 @@ class RsInferenceContext(
     val items: StdKnownItems
 ) : RsInferenceData {
     val fulfill: FulfillmentContext = FulfillmentContext(this, lookup)
-    private val bindings: MutableMap<RsPatBinding, Ty> = HashMap()
+    val bindings: MutableMap<RsPatBinding, Ty> = HashMap()
```

Comment:
There is already a function `getBindingType`, so it's not necessary to make this field public...